docs/source/methods/neutron_physics.rst

```diff
@@ -359,7 +359,7 @@ secondary energy and angle sampling.
 For a reaction with secondary products, it is necessary to determine the
 outgoing angle and energy of the products. For any reaction other than elastic
 and level inelastic scattering, the outgoing energy must be determined based on
-tabulated or parameterized data. The `ENDF-6 Format <endf102>`_ specifies a
+tabulated or parameterized data. The `ENDF-6 Format` specifies a
```

Comment:
Was this supposed to be a link like it is in the change below?
```suggestion
tabulated or parameterized data. The `ENDF-6 Format`_ specifies a
```

Reply:
Yes, it absolutely was. Thanks for catching this!